doc/memos/rdm-draft-alamos-lanzieri-runtime-configuration-architecture.md

```diff
@@ -0,0 +1,139 @@
+- RDM: rdm-draft-alamos-lanzieri-runtime-configuration-architecture.md
+- Title: Runtime configuration architecture
+- Authors: José Álamos, Leandro Lanzieri
+- Status: draft
+- Type: Design
+- Created: December 2018
+
+## Abstract
+This memo describes the proposed high-level architecture and
+mechanisms to implement a runtime configuration system on a RIOT node.
+
+A runtime configuration system is in charge of providing a mechanism to set and
+get the values of configuration parameters that are used during the execution of the firmware,
+as well as a way to persist these values. Most of times these values are
+per-node specific.
```

Comment:
Unclear - what does "most of the time" mean here? I would assume that "runtime configurations are deployment-specific and can be changed on a per node basis. Appropriate management tools could also enable the configuration of node groups."

Reply:
That's true. Will change